examples/python/native/test_gather.py

```diff
@@ -0,0 +1,35 @@
+from flexflow.core import *
```

Comment:
Very small comment, but should we rename this file to avoid using the word "test" if it's just an example? If, on the other hand, this is a test, should we move it to the `tests` folder to avoid crowding the examples folder too much?

Reply:
Fixed.